python/lsst/sims/utils/fileMaps.py

```diff
@@ -17,11 +18,13 @@ def __setitem__(self, key, val):
         self.D[key] = val
 
     def __getitem__(self, item):
-        item = item.rstrip()
+        item = item.rstrip().lstrip()
         if self.D.has_key(item):
             return self.D[item]
         for key, val in sorted(self.subdir_map.iteritems()):
             if re.match(key, item):
+                if item[-3:] == '.gz':
```

Comment:
Oh no! The new sedfilenames in the mlt tables are the only ones to  end in .gz. That's too bad.  Only comment here is that the string method is .endswith() can make your code read more like english:
`full_name = item if item.endswith('.gz') else item + 'gz'`
`return os.path.join(val, full_name)`